kong/api/routes/health.lua

```diff
@@ -42,15 +39,35 @@ return {
 
       -- nginx stats
 
-      local r = ngx.location.capture "/nginx_status"
-      if r.status ~= 200 then
-        kong.log.err(r.body)
-        return kong.response.exit(500, { message = "An unexpected error happened" })
+      if ffi.arch == "x64" or ffi.arch == "arm64" then
+        ffi.cdef[[
+        uint64_t *ngx_stat_requests;
+        uint64_t *ngx_stat_accepted;
+        uint64_t *ngx_stat_handled;
+        uint64_t *ngx_stat_active;
+        uint64_t *ngx_stat_reading;
+        uint64_t *ngx_stat_writing;
+        uint64_t *ngx_stat_waiting;
+        ]]
+      elseif ffi.arch == "x86" or ffi.arch == "arm" then
```

Comment:
```suggestion

      elseif ffi.arch == "x86" or ffi.arch == "arm" then
```

Need a newline before `elseif`.

Reply:
addressed in latest commit [2da37f0](https://github.com/Kong/kong/pull/8690/commits/2da37f028c929829df7c2ca77b70b5a08bda408d)